pages/my-nfts/[chainAccount].tsx

```diff
@@ -89,16 +90,12 @@ const Collection = ({ chainAccount }: Props): JSX.Element => {
   useEffect(() => {
     if (currentUser) {
```

Comment:
Can we consolidate this into an `if/else` statement?

```js
if (currentUser && chainAccount === currentUser.actor) {
  setCurrentProfile('');
} else {
  setCurrentProfile(capitalize(chainAccount));
}
```

Reply:
I've adjusted the if statement to make it easier to read, but I don't think it's necessary to swap out the others.